app/services/doc_auth/error_generator.rb

```diff
@@ -210,8 +218,13 @@ def scan_for_unknown_alerts(response_info)
       unknown_fail_count
     end
 
-    def self.wrapped_general_error
-      { general: [Errors::GENERAL_ERROR], hints: true }
+    def self.general_error(_liveness_enabled)
+      # May have different general error for liveness or non-liveness
```

Comment:
If we're not sure about this yet, should we leave the general error as is, until we determine we need to change it depending on liveness vs non-liveness?

Reply:
@night-jellyfish , yes design will work on error messages, for a POC, this is restored from past code, where a different general error exits when livneness check is used.

Reply:
Ok. I wasn't sure if we'd want this in the code until we're ready for the change.

Reply:
@night-jellyfish , removed comment, it will be the value of config flag.